api/app/Models/User.php

```diff
@@ -618,18 +668,18 @@ public static function scopeEquity(Builder $query, ?array $equity): Builder
         return $query;
     }
 
-    public static function scopeGeneralSearch(Builder $query, ?string $search): Builder
+    public static function scopeGeneralSearch(Builder $query, ?array $searchTerms): Builder
     {
-        if ($search) {
-            $query->where(function ($query) use ($search) {
-                self::scopeName($query, $search);
-                $query->orWhere(function ($query) use ($search) {
-                    self::scopeEmail($query, $search);
-                });
-                $query->orWhere(function ($query) use ($search) {
-                    self::scopeTelephone($query, $search);
-                });
-            });
+        if ($searchTerms && is_array($searchTerms)) {
+            $combinedSearchTerm = implode('&', array_map('trim', $searchTerms));
+            $resultIds = self::search($combinedSearchTerm)->usingWebSearchQuery()
```

Comment:
Does the Scout search function accept something like `take(100)` as well?

Reply:
I just realized that [this comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8539#discussion_r1434665504) addresses this.